client/components/Navigation.js

```diff
@@ -15,15 +16,24 @@ import logoUrl from '../images/CityOfSanJose_logo.png';
  * @param {string} props.path the path for the nav link
  * @param {*} props.children the component children
  */
-let NavLink = ({ location, path, children }) => (
-  <Link
-    to={path}
-    className="c_navigation__link"
-    disabled={location.pathname === path}
-  >
-    {children}
-  </Link>
-);
+let NavLink = ({ location, path, children }) => {
+  const isActive = location.pathname === path;
+  const className = 'c_navigation__link';
+  return (
+    <Link
+      to={path}
+      className={classnames(
```

Comment:
can we clean this up a little? Took a minute to understand

Reply:
Can you elaborate a bit about what was difficult to understand? Is it the use of `classnames()`?

Reply:
its mainly just the nestedness when you could declare a const and then just apply but this is me being picky.